tests/vesting/Vesting.js

```diff
@@ -41,12 +43,15 @@ const VestingRegistryProxy = artifacts.require("VestingRegistryProxy");
 const MAX_DURATION = new BN(24 * 60 * 60).mul(new BN(1092));
 const WEEK = new BN(7 * 24 * 60 * 60);
 
-const TOTAL_SUPPLY = "10000000000000000000000000";
+const TOTAL_SUPPLY = "20000000000000000000000000";
 const ONE_MILLON = "1000000000000000000000000";
+const FOUR_WEEKS = 2419200;
```

Comment:
pls remove unused variable

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)